llorens/day01/Day01Test.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import day01.Day01Solver as solver
+import AoCReader
+
+
+class TestDay01(unittest.TestCase):
+
+    def test_solver_part_1(self):
```

Comment:
I know that you never think in pragmatism and good practices, you are a f*** crazy computer science algorithm solver 2k17. 

But, I recommend you that not put all assert in the same method because when it fails you don't get a fast feedback from the test output about what was wrong, **except** for example in python, you have a third parameter in assertEquals to put a description about your assert and when it fails you can see quickly the test feedback.

For example, `self.assertEqual(solver.solve_part1("1122"), 3, "1122 should be 3 (1 + 2)")`.

Reply:
Wow didn't know that! I'll do so!